cimr/processor/process.py

```diff
@@ -588,9 +637,9 @@ def process_file(self):
             # default behavior will push all missing columns to last
             # delim_whitespace=True,
             # sep='\t| ',
-            sep='\t',
+            sep=self.sep,
             header=0,
-            engine='python',
+            # engine='python',
```

Comment:
Are you going to use this keyword argument in the future? If not, you might as well remove it instead of commenting it out.